storage_service/locations/models/space.py

```diff
@@ -261,6 +261,28 @@ def delete_path(self, delete_path, *args, **kwargs):
         except AttributeError:
             return self._delete_path_local(delete_path)
 
+    def move_direct(self, source_path, destination_path, destination_space, *args, **kwargs):
+        """
+        Move self.path/source_path direct to destination_space.path/destination_path bypassing staging.
+        """
+        LOGGER.debug('move_direct: source_path: %s', source_path)
+        LOGGER.debug('move_direct: destination_path: %s', destination_path)
+        LOGGER.debug('move_direct: destination_space.path: %s', destination_space.path)
+
+        source_path = os.path.join(self.path, source_path)
+
+        if os.path.isabs(destination_path):
+            destination_path = destination_path.lstrip(os.sep)
+
+        destination_path = os.path.join(destination_space.path, destination_path)
+        LOGGER.debug('move_direct: absolute destination_path: %s', destination_path)
+
+        try:
+            self.get_child_space().move_direct(
```

Comment:
`Space.move_direct` needs to return the return value of `self.get_child_space().move_direct()` and not just call it; this is required by `storage_effects = v.src_space.move_direct` in models/package.py where, as things stand, `storage_effects` will be `None`.

Reply:
Ah thanks! We'll take a closer look at the other usages/implementers of the move_to_storage_service/move_from_storage_service to make we haven't missed anything else like that.